### DISCUSSION TITLE
Question about ByteTrack usage

### DISCUSSION THREAD

**Discussion by sholotiuk** [roboflow/supervision#29] [Q&A]
I searched for a solution for people detection and counting and found your amazing work! I mean this one: https://github.com/roboflow/notebooks/blob/main/notebooks/how-to-track-and-count-vehicles-with-yolov8.ipynb

But I need to get the total count of people who appeared during the video. For that, I decided to count all unique track_ids but the problem is the count of track_ids is more than the real number of people during the video.

Here is my notebook(the solution is in the last cell):
https://colab.research.google.com/drive/1m80NFcOanEv1jYkqzvnChs59tvRYKW0d?usp=sharing

Here is the link to the output video where you can see the incorrect wort of the counter:
https://drive.google.com/file/d/1-6V4Ac3tKThQzVgBWDVpmSzlWZIzqIqC/view?usp=share_link

I would be so grateful if you could check my notebook and give me some feedback or advice about that problem!

**Comment by SkalskiP** [comment]
Hi, @sholotiuk 👋🏻! Thanks a lot for moving the conversation here. I wanted to do it for two reasons:
- Most of my work happens on GH, and I'm terrible at responding to emails
- Any solution to your problem might just be a solution to a problem that others experience

First things first, could you grant me access to your notebook? `piotr@roboflow.com` is my email.

**Reply by sholotiuk** [reply]
Hi Piotr!
Sure, done! Hope it works now

**Reply by mehdidouyeb** [reply]
Hey any news on this topic  ?

**Reply by rolson24** [reply]
Hey @mehdidouyeb
I think the best way to count the number of people in a video is to use ByteTrack and then setup a line counter to count how many people cross the line. You can set up a few lines to capture the common movement paths. This is shown in the colab notebook from OP I think.
